nucypher/cli/characters/ursula.py

```diff
@@ -121,17 +122,22 @@ def ursula(click_config,
 
     """
 
+    # Validate
+    if federated_only:
+        raise click.BadOptionUsage(option_name="--geth", message="Federated only cannot be used with the --geth flag")
+
+    if click_config.debug and quiet:
+        raise click.BadOptionUsage(option_name="quiet", message="--debug and --quiet cannot be used at the same time.")
+
```

Comment:
Is `--lonely` and `--teacher-uri` at odds with each other as well like above?

Reply:
Yeah - I think its a good Idea to crash if both flags are passed.

Reply:
Issue #973